### PR TITLE
[1LP][RFR] Add failsafe delete in case of TimeOutError

### PR DESCRIPTION
**Pull request by jistefan**
Purpose or Intent
=================
__Extending__ the tests for case of timeout.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a263bfa954990a00aaf6e9fd476783f3edf3a45e:

`cfme/tests/cloud/test_keypairs.py`:
- :bangbang: Line 52:5: **E303** *too many blank lines (2)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/cloud/test_keypairs.py [hunk 1]
```diff
@@ -41,7 +42,12 @@ def test_keypair_crud(openstack_provider):
     view = navigate_to(keypair, 'Details')
     assert view.is_displayed
 
-    keypair.delete(wait=True)
+    try:
+        keypair.delete(wait=True)
+    except TimedOutError:
+        get_mgmt(openstack_provider.name).api.keypairs.delete(keypair.name)
```
Comment (izapolsk) [inline cfme/tests/cloud/test_keypairs.py]: REQUIRED: please use 
openstack_provider.mgmt.api blabla instead.

Comment (jistefan) [inline cfme/tests/cloud/test_keypairs.py]: Helllo, corrected

**Comment by jistefan** [comment]
@izapolsk seems like PRT has risen from underground. Please merge ;)

**Pull request merged**